functions/src/monitor/monitor.ts

```diff
@@ -80,15 +80,31 @@ export const ping: any = async (projectUid: string, monitorUid: string): Promise
     .set(pingResult);
 }
 
-export const deleteMonitorPings: any = async (projectUid: string, monitorUid: string): Promise<WriteResult[]> => {
+export const deleteMonitorPings: any = async (projectUid: string, monitorUid: string, isExpiredPings: boolean): Promise<WriteResult[]> => {
```

Comment:
I think in the future, we should change `isExpiredPings: boolean` to `isExpiredPings: number`, because the projects will have different grades in the future (something like `free` and `pro`), where free we will only keep for a short amount of time and pro we keep for longer. This is functionality where we pass in different expire time for pings I think we can easily add in the future 🤓

Reply:
Do you want this change in this PR or later ?

Reply:
Later. It is not something we need now, I am just thinking out loud 😄

Reply:
Great 👍